docker-compose.apps.yml

```
@@ -149,6 +149,37 @@ services:
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+      - "2181:2181"
+
+  kafka:
+    image: confluentinc/cp-kafka:latest
+    depends_on:
+      - zookeeper
+    profiles:
+      - infra
+    healthcheck:
+      test: [ "CMD", "kafka-topics", "--bootstrap-server", "localhost:9092", "--list" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)